src/measure_theory/simple_func_dense.lean

```diff
@@ -168,60 +208,147 @@ begin
   exact_mod_cast this,
 end
 
-lemma tendsto_approx_on_L1_edist  [opens_measurable_space E]
- {f : β → E} (hf : measurable f) {s : set E} {y₀ : E} (h₀ : y₀ ∈ s) [separable_space s]
-  {μ : measure β} (hμ : ∀ᵐ x ∂μ, f x ∈ closure s) (hi : has_finite_integral (λ x, f x - y₀) μ) :
-  tendsto (λ n, ∫⁻ x, edist (approx_on f hf s y₀ h₀ n x) (f x) ∂μ) at_top (𝓝 0) :=
+lemma tendsto_approx_on_Lp_snorm [opens_measurable_space E]
+  {f : β → E} (hf : measurable f) {s : set E} {y₀ : E} (h₀ : y₀ ∈ s) [separable_space s]
+  (hp_ne_zero : p ≠ 0) (hp_ne_top : p ≠ ⊤) {μ : measure β} (hμ : ∀ᵐ x ∂μ, f x ∈ closure s)
```

Comment:
usually we write `hp_ne_top` as you do, but we use the symbol `∞` instead of `⊤` (here and below).